code/modules/projectiles/gun.dm

```diff
@@ -303,7 +303,7 @@
 				to_chat(user, "<span class='warning'>[src] is lethally chambered! You don't want to risk harming anyone...</span>")
 				return
 		if(randomspread)
-			sprd = round((rand() - 0.5) * DUALWIELD_PENALTY_EXTRA_MULTIPLIER * (randomized_gun_spread + randomized_bonus_spread))
+			sprd = round((rand(0, 1) - 0.5) * DUALWIELD_PENALTY_EXTRA_MULTIPLIER * (randomized_gun_spread + randomized_bonus_spread))
```

Comment:
Eh? `rand()` is already `rand(0,1)`.

Reply:
There's a key difference between 0,1 and between 0 and 1. One goes by integers, the other by floats
![image](https://user-images.githubusercontent.com/58055496/114660104-9baacc00-9ca9-11eb-81b8-acfec3a7388c.png)